source/reference/verification/verifier.txt

```diff
@@ -0,0 +1,112 @@
+.. _c2c-migration-verifier:
+
+==============================
+Verify with Migration Verifier
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Migration Verifier connects to the source and destination clusters and performs
+a series of verification checks, comparing documents, views, and indexes to
+confirm the sync was successful.
+
+
+About This Task
+---------------
+
+Migration Verifier is an experimental and unsupported tool.
+
+For installation instructions, see
+`GitHub <https://github.com/mongodb-labs/migration-verifier>`__.
+
+Steps
+-----
+
+.. procedure::
+
+   .. step:: Start Migration Verifier
```

Comment:
I think migration verifier should be started once mongosync reaches CEA, not before.

Reply:
Fixed